pkg/util/ansible_execution.go

```diff
@@ -122,6 +122,7 @@ func AnsibleExecution(
 			ansibleEE.Spec.ExtraVars["edpm_override_hosts"] = json.RawMessage([]byte(fmt.Sprintf("\"%s\"", targetNodeset)))
 			util.LogForObject(helper, fmt.Sprintf("for service %s, substituting existing ansible play host with '%s'.", service.Name, targetNodeset), ansibleEE)
 		}
+		ansibleEE.Spec.ExtraVars["edpm_service_name"] = json.RawMessage([]byte(fmt.Sprintf("\"%s\"", service.Name)))
```

Comment:
so this should have envtest coverage at a minium
possible also kuttl at some point but envtest should be added to make sure this does not regress in the future.

if we add that, that is probably the minium test coverage required to merge this but we should also enable tls in in the zuul job possible as a separate pr

Reply:
i think https://github.com/openstack-k8s-operators/dataplane-operator/pull/754 will enable it by default and as a result enable it in the ci jobs